### PR TITLE
Activities @ Mississauga

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description

Explore activities for the city of Mississauga, Ontario, Canada 🇨🇦 

## Screencast
active-mississaugua-1
active-mississaugua-4
active-mississaugua-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/active-mississaugua/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "icon": "command-icon.png",
+  "author": "Aayush9029",
+  "categories": [
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Registered Programs",
```
Comment (danielsequeira) [inline extensions/active-mississaugua/package.json:15]:
```suggestion
      "title": "Search Registered Programs",
```

File: extensions/active-mississaugua/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "subtitle": "Explore Activities",
```
Comment (danielsequeira) [inline extensions/active-mississaugua/package.json:16]:
```suggestion
      "subtitle": "Active Mississaugua",
```

File: extensions/active-mississaugua/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      "description": "Register for Culture, Library and Recreation programs/activities. Like fitness, swimming, skating, camps, sports etc.",
+      "mode": "view"
+    },
+    {
+      "name": "dropin",
+      "title": "Dropin Programs",
```
Comment (danielsequeira) [inline extensions/active-mississaugua/package.json:22]:
```suggestion
      "title": "Search Dropin Programs",
```

File: extensions/active-mississaugua/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      "subtitle": "Walk-in Activities",
```
Comment (danielsequeira) [inline extensions/active-mississaugua/package.json:23]:
```suggestion
      "subtitle": "Active Mississaugua",
```

File: extensions/active-mississaugua/package.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      "description": "Drop-in programs fill on a first come, first served basis until full. Must be in-person to secure your space.",
```
Comment (danielsequeira) [inline extensions/active-mississaugua/package.json:24]:
```suggestion
      "description": "Drop In programs fill on a first come, first served basis until full. Must be in-person to secure your space.",
```

File: extensions/active-mississaugua/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Active Mississaugua Changelog
+
+## [Initial Version] - 2022-07-25
```
Comment (danielsequeira) [inline extensions/active-mississaugua/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - 2022-08-03
```

**Comment by danielsequeira** [comment]
Hi @Aayush9029,

Thank you for another contribution, just a few nitpicking comments from my side 🙏 

Besides that it seems that the extension icon's resolution is not the best, could you find a better one?

**Comment by Aayush9029** [comment]
I whipped out a quick one using SVG hopefully it'll look a tad bit better

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**